user_manual/talk/talk_basics.rst

```diff
@@ -104,7 +141,18 @@ You can hide your video (useful during a screen share) with the little arrow jus
 
 You can access your settings and choose a different webcam, microphone and other settings in the ``...`` menu in the top bar.
 
-.. image:: images/call-menu.png
+.. image:: images/media-settings.png
+    :width: 300px
+
+.. image:: images/device-settings-before-call.png
```

Comment:
You should put that image before `images/join-call.png`, and set a small description, i think:

```
When you're in a conversation, you can start a call any time with the ``Start call`` button. Other participants will get notified and can join the call. Before call is started, you can select and switch on/off media devices (webcam, microphone) and select a virtual background


.. image:: images/device-settings-before-call.png
:width: 300px

 If somebody else has started a call already, the button will change in a green ``Join call`` button.
.. image:: images/join-call.png
```